packages/track-transcript/src/TranscriptTrack.js

```diff
@@ -93,37 +93,25 @@ const TranscriptName = styled.div`
 `
 
 const GtexTitleWrapper = styled.div`
+  box-sizing: border-box;
```

Comment:
This element has `width: 100%` and `padding-left: 10px`, so with the default `box-sizing: content-box`, it extended 10px outside of the right panel.